### PR TITLE
LPS-197065 fix errors in item selector and sharepoint interaction

### PR DESCRIPTION
**Pull request by robertoDiaz**

### CODE REVIEW

File: modules/dxp/apps/sharepoint-rest/sharepoint-rest-repository/src/main/java/com/liferay/sharepoint/rest/repository/internal/document/library/repository/external/SharepointRepositoryDefiner.java [hunk 1]
```diff
@@ -83,6 +84,9 @@ public void registerCapabilities(
 				ProcessorCapability.ResourceGenerationStrategy.
 					ALWAYS_GENERATE));
 
+		capabilityRegistry.addExportedCapability(
+			ExtRepositoryCapability.class, new ExtRepositoryCapability());
+
```
Comment (adolfopa) [inline modules/dxp/apps/sharepoint-rest/sharepoint-rest-repository/src/main/java/com/liferay/sharepoint/rest/repository/internal/document/library/repository/external/SharepointRepositoryDefiner.java:89]: This shouldn't be needed. Check usages of `RepositoryUtil.isExternalRepository`.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/sharepoint-rest/sharepoint-rest-repository/src/main/java/com/liferay/sharepoint/rest/repository/internal/document/library/repository/external/SharepointRepositoryDefiner.java (hunk 1 above)
Comment (robertoDiaz) [reply modules/dxp/apps/sharepoint-rest/sharepoint-rest-repository/src/main/java/com/liferay/sharepoint/rest/repository/internal/document/library/repository/external/SharepointRepositoryDefiner.java]: Fixed. (didn't know abut this method)

That's exactly what I needed.

Comment (robertoDiaz) [reply modules/dxp/apps/sharepoint-rest/sharepoint-rest-repository/src/main/java/com/liferay/sharepoint/rest/repository/internal/document/library/repository/external/SharepointRepositoryDefiner.java]: Fixed, I didn't remember that tool.

That's exactly what I need :)

**Comment by robertoDiaz** [comment]
ci:test:sf

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**